evm_arithmetization/src/cpu/kernel/asm/account_code.asm

```diff
@@ -139,3 +139,66 @@ remove_padding_loop:
 remove_padding_after:
     %stack (offset, ctx, retdest) -> (retdest, offset)
     JUMP
+
+// Convenience macro to call poseidon_hash_code_unpadded and return where we left off.
+%macro poseidon_hash_code_unpadded
+    %stack (addr, len) -> (addr, len, %%after)
```

Comment:
Nit: requires that the address provided has an offset of 0. It might be useful to precise it here.